crates/core_arch/src/x86/sse.rs

```diff
@@ -1938,6 +1938,56 @@ extern "C" {
 /// exception _may_ be generated.
 ///
 /// [Intel's documentation](https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm_stream_ps)
+///
+/// # Safety of non-temporal stores
```

Comment:
Minor style thing: putting the full non-temporal store info on `_mm_sfence`, which is necessarily referenced from the short form docs, probably makes more sense than choosing `_mm_stream_ps` as a "most canonical" non-temporal store to document this on.

Reply:
Okay, I've moved this to the fence docs.